root/static/scripts/release-editor/utils.js

```diff
@@ -153,6 +157,12 @@ function cleanArtistCreditName(data) {
     };
 }
 
+function cleanIsrc(data) {
+    return {
+        isrc: data.id,
```

Comment:
This should just be
```suggestion
        isrc: data,
```
since the JSON format doesn't have any id property like the XML, it's just a list of strings